builtin/files/userdata/cloud-config-controller

```diff
@@ -1058,6 +1058,9 @@ write_files:
         "${mfdir}/kube-dns-de.yaml"
       {{- end }}
       {{ if .KubeDns.NodeLocalResolver -}}
+      {{ if .KubeDns.dnsmasq.CoreDNSLocal.Enabled -}}
+      deploy "${mfdir}/dnsmasq-node-coredns-local.yaml"
```

Comment:
I know that `kube-aws` doesn't do a fantastic job of clean up, but seems like this one would be easy. Could we throw in a 

```
{{- else }}
      remove "${mfdir}/canal.yaml"
{{- end }}}
```

Reply:
Sorry, `remove "${mfdir}/dnsmasq-node-coredns-local.yaml"`

Reply:
👍 Will do.